docs/development/SETUP.md

```diff
@@ -62,6 +62,24 @@ VEDA-UI includes a GitHub action for checking TypeScript and lint errors. If you
 #### Pre-commit Hook
 Additionally, there's a pre-commit hook that performs the same error checks. This helps developers identify and address issues at an earlier stage. If you need to bypass the check (to make a local temporary commit etc.), include the `--no-verify`` flag in your commit command.
 
+#### Format on Save
+While the existing checks ensure that no linting errors are introduced into the main branch, enabling "Format on Save" in your code editor provides an additional layer of convenience. It helps catch formatting issues early in the development process, minimizing the risk of committing errors and ensuring consistent code style across the team.
+
+##### How to Set Up Format on Save in VS Code
+Install Required Extensions:
+
+1. Install the ESLint extension ("dbaeumer.vscode-eslint").
+2. Enable Format on Save in the settings (Cmd+, on macOS)
```

Comment:
Is the shortcut for settings `Cmd+` ? Mine shows that it is `Cmd,` !
<img width="495" alt="Screenshot 2024-12-11 at 11 28 58 AM" src="https://github.com/user-attachments/assets/dad564f2-96f1-48d3-9a21-3fd4d721f742" />

Reply:
That actually means `Cmd` + `,`, so the same as yours. 
This should make it easier to read:
```suggestion
2. Enable Format on Save in the settings (<kbd>⌘ Command</kbd> + <kbd>,</kbd> on macOS)
```

Reply:
That looks great!

<img width="141" alt="Screenshot 2024-12-12 at 10 11 53" src="https://github.com/user-attachments/assets/a5d8f731-9627-430f-ae49-5762041aebdd" />

Reply:
ohh haha, sorry for. misreading! Thanks for making it clear!